### ISSUE TITLE
[Policy Assistant] explain command: support setting (B)ANPs via CLI arg

### ISSUE THREAD

**Issue by huntergregory** [kubernetes-sigs/network-policy-api#201]
Successor to #153. Child issue of #150.

Right now we have AdminNetworkPolicies hard-coded (referenced in analyze.go). A user should be able to specify NetworkPolicy, AdminNetworkPolicy, and the BaselineAdminNetworkPolicy by:
- grabbing these from their Kubernetes cluster
- specifying the filepath for all policy yamls
- or both

Code in question:
https://github.com/kubernetes-sigs/network-policy-api/blob/669dfbc72245a2c7c6f3a3dd5c6188b7af9f8014/cmd/policy-assistant/pkg/cli/analyze.go#L96-L128

**Comment by Peac36** [comment]
Hey, @huntergregory can you share more details about the task? Is it about reading (B)ANPs from a cluster or file path?
Since I'm new to the project any information would be helpful.

---

Is there a weekly meeting about this project where I can join and ask these kinds of questions?

**Comment by huntergregory** [comment]
Hey Nikola, thanks for looking to pick this up. Just fixed this issue's description, let me know if you have any questions.

> Is there a weekly meeting about this project where I can join and ask these kinds of questions?

Yes! Would be great to have you. We meet every other Tuesday (some more details at [this link](https://docs.google.com/document/d/1AtWQy2fNa4qXRag9cCp5_HsefD7bxKe3ea2RPn8jnSs/edit?usp=sharing))

**Comment by huntergregory** [comment]
I think the CLI already has the arguments we need (code is in analyze.go). I think they are like ‘--policy-path’, ‘--namespace’, ‘--all-namespaces’. 

To create the CLI binary:
- change into the cmd/policy-assistant/ directory
- `make cyclonus`
- binary at ‘cmd/cyclonus/cyclonus’ I believe

**Comment by Peac36** [comment]
Thanks for the info Hunter.I already started working on this, but more questions arose.

Should cyclonus always work with the (B)ANPs or they should be enabled/disabled via an arguments?

What is the expected behavior if one of the request fail while trying to get info from the api-server?

**Comment by huntergregory** [comment]
> Should cyclonus always work with the (B)ANPs or they should be enabled/disabled via an arguments?
Let's always include every policy type.

> What is the expected behavior if one of the request fail while trying to get info from the api-server?
I would just log errors with `logrus.Errorf()` and continue. But if API Server returns this:
```
error: the server doesn't have a resource type "adminnetworkpolicy"
```
ideally, we would just log a warning message that ANP isn't defined instead of saying there's an error.

**Comment by huntergregory** [comment]
Fixed in #239
